viewing_party/party.py

```diff
@@ -1,23 +1,156 @@
 # ------------- WAVE 1 --------------------
 
+from tests.test_constants import MOVIE_TITLE_1
+
+
 def create_movie(title, genre, rating):
-    pass
+    movie_dict = {}
+    if not title:
+        return None
+    elif not genre:
+        return None
+    elif not rating:
+        return None

```

Comment:
This works great! Consider making this a bit more compact by using a single compound conditional.